libs/database/src/lib/services/notification.service.ts

```diff
@@ -499,22 +391,135 @@ export class NotificationService {
   };
 
   checkNotify = (notify) => {
-    if (notify.type === NotificationType.Like) {
-      return Environment.NOTIFY_LIKE == '1' ? true : false;
-    } else if (notify.type === NotificationType.Recast) {
-      return Environment.NOTIFY_RECAST == '1' ? true : false;
-    } else if (notify.type === NotificationType.Quote) {
-      return Environment.NOTIFY_QUOTE == '1' ? true : false;
-    } else if (notify.type === NotificationType.Comment) {
-      return Environment.NOTIFY_COMMENT == '1' ? true : false;
-    } else if (notify.type === NotificationType.Farm) {
-      return Environment.NOTIFY_FARM == '1' ? true : false;
-    } else if (notify.type === NotificationType.Reply) {
-      return Environment.NOTIFY_REPLY == '1' ? true : false;
-    } else if (notify.type === NotificationType.Tag) {
-      return Environment.NOTIFY_TAG == '1' ? true : false;
-    } else {
-      return Environment.NOTIFY_SYSTEM == '1' ? true : false;
+    let isNotify = false;
+    switch (notify.type) {
+      case NotificationType.Like:
+        isNotify = Environment.NOTIFY_LIKE == '1' ? true : false;
+        break;
+
+      case NotificationType.Recast:
+        isNotify = Environment.NOTIFY_RECAST == '1' ? true : false;
+        break;
+
+      case NotificationType.Quote:
+        isNotify = Environment.NOTIFY_QUOTE == '1' ? true : false;
+        break;
+
+      case NotificationType.Comment:
+        isNotify = Environment.NOTIFY_COMMENT == '1' ? true : false;
+        break;
+
+      case NotificationType.Farm:
+        isNotify = Environment.NOTIFY_FARM == '1' ? true : false;
+        break;
+
+      case NotificationType.Reply:
+        isNotify = Environment.NOTIFY_REPLY == '1' ? true : false;
+        break;
+
+      case NotificationType.Tag:
+        isNotify = Environment.NOTIFY_TAG == '1' ? true : false;
+        break;
+
+      default:
+        isNotify = Environment.NOTIFY_SYSTEM == '1' ? true : false;
+        break;
     }
+    return isNotify;
```

Comment:
```suggestion
    switch (notify.type) {
      case NotificationType.Like:
        return Environment.NOTIFY_LIKE == '1'
      case NotificationType.Recast:
        return Environment.NOTIFY_RECAST == '1'
      case NotificationType.Quote:
        return Environment.NOTIFY_QUOTE == '1'
      case NotificationType.Comment:
        return Environment.NOTIFY_COMMENT == '1'
      case NotificationType.Farm:
        return Environment.NOTIFY_FARM == '1'
      case NotificationType.Reply:
        return Environment.NOTIFY_REPLY == '1'
      case NotificationType.Tag:
        return Environment.NOTIFY_TAG == '1'
      default:
        return Environment.NOTIFY_SYSTEM == '1'
    }
```

switch + return ดีกว่า การ reassign + break นะ ผมว่า